<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultisigDialog</class>
 <widget class="QDialog" name="MultisigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>753</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1054</width>
    <height>753</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Multisignature Address Interactions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QTabWidget" name="multisigTabWidget">
     <property name="minimumSize">
      <size>
       <width>1036</width>
       <height>727</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea{border: 1px solid #5b4c7c;}
QFrame{background-color:#f2f0f0;}
QLabel{background-color:#ffffff;}
QFrame &gt; QLabel{background-color:#f2f0f0;}</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="addMultisigTab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Create MultiSignature &amp;Address</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="enterMLayout">
         <item>
          <widget class="QSpinBox" name="numsigs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>How many people must sign to verify a transaction</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>16</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="enterMLabel">
           <property name="text">
            <string>Enter the minimum number of signatures required to sign transactions</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="addressLabelLayout"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="addAddressLayout">
         <item>
          <widget class="QLineEdit" name="addpubkey"/>
         </item>
         <item>
          <widget class="QPushButton" name="makeaddr_ADD_PUB_KEY">
           <property name="text">
            <string>add pubkey/address</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget_addr">
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string>pubkey</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>multisig address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="makeaddress"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>                                                      redeemScript</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="addMultisigLayout">
         <item>
          <widget class="QPushButton" name="create_addr">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Create a new multisig address</string>
           </property>
           <property name="toolTipDuration">
            <number>-3</number>
           </property>
           <property name="text">
            <string>C&amp;reate</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/filesave</normaloff>:/icons/filesave</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset_addr">
           <property name="text">
            <string>RESET</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="makeaddrRS">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>75</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="importLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="createTransactionTab">
      <attribute name="title">
       <string>&amp;Create MultiSignature Tx</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>INPUTS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="addscriptkey">
             <property name="maximumSize">
              <size>
               <width>316</width>
               <height>22</height>
              </size>
             </property>
             <property name="whatsThis">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;script key of input in the HEX form&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>script key of input in the HEX form</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="V_OUT">
             <property name="minimumSize">
              <size>
               <width>133</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>133</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>V OUT</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addinput">
             <property name="minimumSize">
              <size>
               <width>121</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>121</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>OUTPUTS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="addOutput">
             <property name="maximumSize">
              <size>
               <width>316</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>add Output address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="amount">
             <property name="maximumSize">
              <size>
               <width>316</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>amount</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_output">
             <property name="minimumSize">
              <size>
               <width>121</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>121</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QTreeWidget" name="tree_wiget_create_raw">
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string>v out</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>input</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="tree_wiget_create_raw_output">
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string>amount</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>payTo</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="clear_in">
           <property name="text">
            <string>Clear inputs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset">
           <property name="text">
            <string>Rest</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear_out">
           <property name="text">
            <string>Clear outputs</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="makerawtx"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="send">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Create a transaction object using the given inputs to the given outputs</string>
         </property>
         <property name="text">
          <string>make raw TX</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/export</normaloff>:/icons/export</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="signMultisigTransaction">
      <property name="styleSheet">
       <string notr="true">txScrollArea:{
	border: 1px solid #5b4c7c;
}
keyScrollArea:{
	border: 1px solid #5b4c7c;
}
txScrollArea:{
	border: 1px solid #5b4c7c;
}</string>
      </property>
      <attribute name="title">
       <string>&amp;Sign MultiSignature Tx</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLineEdit" name="hash">
           <property name="text">
            <string>hash</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PK">
           <property name="text">
            <string>private key</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0" rowspan="3">
          <widget class="QTextEdit" name="sign_hex">
           <property name="maximumSize">
            <size>
             <width>1012</width>
             <height>201</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="send_TX">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="sign">
           <property name="text">
            <string>sign</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="sig_reset">
           <property name="text">
            <string>reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
